app/main.py

```diff
@@ -1 +1,11 @@
-# write your code here
+import os
+
+
+def move_file(command: str) -> None:
+    data = command.split()
+    if data[0] == "mv" and len(data) == 3 and data[1] != data[2]:
```

Comment:
```suggestion
    if len(data) == 3 and data[0] == "mv" and data[1] != data[2]:
```
first it is better to check that the length is three